external/pxl_documentation.json

```diff
@@ -855,125 +854,366 @@
+            ]
+          },
+          {
+            "ident": "ttl",
+            "desc": "The length of time that a tracepoint will stay alive, after which it will be removed.",
+            "types": [
+              "px.Duration"
+            ]
+          }
+        ]
+      }
     },
     {
       "body": {
-        "name": "DataFrame.filter",
-        "brief": "Returns a DataFrame with only those rows that match the condition.",
-        "desc": "Filters for the rows in the DataFrame that match the boolean condition. Will error out if you don't pass in a boolean expression. The functions available are defined in [UDFs](/reference/pxl/udf).     ",
-        "examples": [
+        "name": "pxtrace.DeleteTracepoint",
+        "brief": "Deletes a tracepoint.",
+        "desc": "Deletes the tracepoint with the provided name, should it exist.  ",
+        "examples": []
+      },
+      "funcDoc": {
+        "args": [
           {
-            "value": "```\ndf = px.DataFrame('http_events')\n# Filter for only http requests that are greater than 100 milliseconds\ndf = df[df['resp_latency_ns'] \u003e 100 * 1000 * 1000]\n```"
+            "ident": "name",
+            "desc": "The name of the tracepoint.",
+            "types": [
+              "str"
+            ]
           }
         ]
+      }
+    }
+  ],
+  "otelDocs": [
+    {
+      "body": {
+        "name": "px.otel.metric.Gauge",
+        "brief": "Defines the OpenTelemetry Metric Gauge type.",
+        "desc": "Gauge describes how to transform a pixie DataFrame into the OpenTelemetry Metric Gauge type.  ",
+        "examples": []
       },
       "funcDoc": {
         "args": [
           {
-            "ident": "key",
-            "desc": "DataFrame expression that evaluates to a bool. Filter keeps any row that causes the expression to evaluate to True.",
+            "ident": "name",
+            "desc": "The name of the metric.",
             "types": [
-              "ScalarExpression"
+              "string"
+            ]
+          },
+          {
+            "ident": "value",
+            "desc": "The column that contains the data. Must be either an INT64 or a FLOAT64.",
+            "types": [
+              "Column"
+            ]
+          },
+          {
+            "ident": "description",
+            "desc": "A description of what the metric tracks.",
+            "types": [
+              "string, optional"
+            ]
+          },
+          {
+            "ident": "attributes",
+            "desc": "A mapping of attribute name to the column name that stores data about the attribute.",
+            "types": [
+              "Dict[string, string], optional"
             ]
           }
         ],
         "returnType": {
-          "desc": "DataFrame with only those rows that return True for the expression.",
+          "desc": "the description of how to map a DataFrame to OpenTelemetry Data. Can be passed into into px.otel.Data() as data.",
```

Comment:
If you fix the other thing upstream, can you change this to add backticks: `px.otel.Data()`?